akvo/rsr/tests/usecases/test_change_project_parent.py

```diff
@@ -85,9 +77,7 @@ def test_change_parent_to_parent_sibling(self):
         # When
         command.change_parent(grand_child.object, child_project2.object)
         # Then
-        parents = grand_child.object.parents_all()
-        self.assertEqual(1, len(parents))
-        self.assertEqual(child_project2.object.id, parents.first().id)
+        self.assertEqual(child_project2.object.id, grand_child.object.parent().id)
```

Comment:
Failed when changed to `self.assertEqual(child_project2.object.id, Project.objects.get(title='Grand child project').parent().id)`

Reply:
Fixed in 086c9ca39ed0327d2c6b33a91903fd69b88318bf